spec/unit/module_spec.rb

```diff
@@ -553,6 +555,70 @@
   it "should return the path to the plugin directory" do
     expect(mod.plugin_directory).to eq(File.join(path, "lib"))
   end
+
+  it "should return the path to the tasks directory" do
+    expect(mod.tasks_directory).to eq(File.join(path, "tasks"))
+  end
+
+  describe "when finding tasks" do
+    before do
+      Puppet::FileSystem.unstub(:exist?)
```

Comment:
I don't actually think we need any of these Puppet::FileSystem.stub/unstub(:exist?) anymore... at least, naively, with them all removed the tests pass. But...another tech debt for another day